include/ftxui/component/screen_interactive.hpp

```diff
@@ -121,6 +121,7 @@ class ScreenInteractive : public Screen {
   bool mouse_captured = false;
   bool previous_frame_resized_ = false;
 
+  bool validated_ = false;
```

Comment:
Why do we need this variable?

I see it can move from false->true on the first drawing, but then it is never reset back to false.

Reply:
I figure out that we do not need this variable in case when clearing appears only after decrease of dimx, so I will remove it. Thanks